plugins/CoreHome/Columns/VisitLastActionDayOfWeek.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\CoreHome\Columns;
+
+use Piwik\Columns\DimensionMetricFactory;
+use Piwik\Columns\MetricsList;
+use Piwik\Plugin\Dimension\VisitDimension;
+use Piwik\Metrics\Formatter;
+
+require_once PIWIK_INCLUDE_PATH . '/plugins/VisitTime/functions.php';
+
+class VisitLastActionDayOfWeek extends VisitDimension
+{
+    protected $columnName = 'visit_last_action_time';
+    protected $type = self::TYPE_DATETIME;
+    protected $segmentName = 'visitEndServerDayOfWeek';
+    protected $nameSingular = 'VisitTime_ColumnVisitEndServerWeek';
```

Comment:
Seems that translation key does not exists. Guess it needs to be `VisitTime_ColumnVisitEndServerDayOfWeek`

Reply:
Cheers... great find. Fixed it. Will have an eye on tests.